go/oasis-test-runner/env/env.go

```diff
@@ -17,6 +22,35 @@ var ErrEarlyTerm = errors.New("env: sub-process exited early")
 // CleanupFn is the cleanup hook function prototype.
 type CleanupFn func()
 
+type ParameterFlagSet struct {
+	flag.FlagSet
+}
+
+// TestInstanceInfo contains information of the current test run.
+type TestInstanceInfo struct {
+	// Test is the name of the test.
+	Test string `json:"test"`
+
+	// Instance is the instance name of the test. e.g. oasis-test-runner123456
+	Instance string `json:"instance"`
+
+	// ParameterSet is the parameter set the test was run with.
+	ParameterSet *ParameterFlagSet `json:"parameter_set"`
+
+	// Run is the number of run.
+	Run int `json:"run"`
+}
+
+func (ifs *ParameterFlagSet) MarshalJSON() (r []byte, err error) {
```

Comment:
Add documentation comment. Also why `ifs` instead of `pfs`?

Reply:
It was called `InstanceFlagSet` before, that's why ifs ;)